core/vmap.h

```diff
@@ -65,7 +65,7 @@ class VMap {
 		const Pair *a = _cowdata.ptr();
 		int middle = 0;
 
-#if DEBUG_ENABLED
+#ifdef DEBUG_ENABLED
```

Comment:
Shouldn't you use #if defined() here if you're using that in the other places?

Reply:
In the places I use `#if defined` is because I like it to match the `#elif`.